core/nodejsActionBase/runner.js

```diff
@@ -22,6 +22,8 @@
  */
 const fs = require('fs');
 const path = require('path');
+const { DynamicPool } = require("node-worker-threads-pool");
+const pool = new DynamicPool(4)
```

Comment:
I'd move the size of the pool `4` to an env var, along side `__OW_*`

Reply:
fixed